test/local-hub/input_validation_test.go

```diff
@@ -180,17 +180,17 @@ func TestExternalStorageInputValidation(t *testing.T) {
 			storageSpec: storageSpec{
 				URL:         ptr.String("http://localhost:5000/fake"),
 				AcceptValue: ptr.Bool(false),
-				ContextSchema: ptr.String(heredoc.Doc(`
+				ContextSchema: []byte(heredoc.Doc(`
 					   yaml: true`)),
 			},
 			value: map[string]interface{}{
 				"name": "Luke Skywalker",
 			},
 			expErrMsg: heredoc.Doc(`
-              while executing mutation to create TypeInstance: All attempts fail:
-              #1: graphql: failed to create TypeInstance: failed to create the TypeInstances: 2 error occurred:
-              	* Error: failed to process the TypeInstance's backend "MOCKED_ID": invalid spec.context: Unexpected token y in JSON at position 0
-              	* Error: rollback externally stored values: failed to process the TypeInstance's backend "MOCKED_ID": invalid spec.context: Unexpected token y in JSON at position 0`),
+			while executing mutation to create TypeInstance: All attempts fail:
+			#1: graphql: failed to create TypeInstance: failed to create the TypeInstances: 2 error occurred:
+				* Error: failed to resolve the TypeInstance's backend "MOCKED_ID": spec.value/contextSchema must be object
+				* Error: rollback externally stored values: failed to resolve the TypeInstance's backend "MOCKED_ID": spec.value/contextSchema must be object`),
```

Comment:
Why do we have `value/contextSchema` in error message?

Reply:
`spec.value` is root path for the `ajv` validator and it is defined [here](https://github.com/capactio/capact/blob/main/hub-js/src/local/storage/service.ts#L431).

Reply:
oh, I thought `/` means `or` in this case, as we have `.` as a delimiter between `spec` and `value`. Can we change the delimiter to have a single one across the full path?
`spec.value.contextSchema` would be the best 👍 

If that's not easily done, then `spec/value/contextSchema` is also fine for me.